ui/app/controllers/jobs/index.js

```diff
@@ -61,11 +61,7 @@ export default Controller.extend(Sortable, Searchable, {
 
   optionsDatacenter: computed('visibleJobs.[]', function() {
     const flatten = (acc, val) => acc.concat(val);
-    const allDatacenters = new Set(
-      this.visibleJobs
-        .mapBy('datacenters')
-        .reduce(flatten, [])
-    );
+    const allDatacenters = new Set(this.visibleJobs.mapBy('datacenters').reduce(flatten, []));
```

Comment:
Maybe this is an extraneous change?

Reply:
As you suspected, this was an autofix. Now that husky is working again this shouldn't happen anymore 💃